### DISCUSSION TITLE
The 2.1.0 update does not adhere to semver?

### DISCUSSION THREAD

**Discussion by AdamGaskins** [topclaudy/compoships#120] [General]
The 2.1.0 update broke my code in production because I was still extending the Model class instead of using the trait. Shouldn't this breaking change have been in a 3.0 update?

**Comment by topclaudy** [comment]
From the README: "Important: Model support was dropped in 2.1. For Model support, use 2.0.5 or below". I suggest to implement the necessary tests to avoid those issues on your production environment.

**Reply by AdamGaskins** [reply]
Yes, that is what I'm referring to. The README says:

> We use SemVer for versioning.

But according to SemVer, changes that are not backwards compatible should increment the major version number. [See spec item 8](https://semver.org/#spec-item-8)

**Reply by topclaudy** [reply]
You're right! This should have been in a 3.x update.

**Comment by topclaudy** [comment]
Model support added back in 2.1.2.
